compiler/testData/diagnostics/tests/controlFlowAnalysis/assignmentInLocalsInConstructor.fir.kt

```diff
@@ -7,7 +7,7 @@ class Test {
         val t = object {
             fun some() {
                 // See KT-13597
-                a = "12"
+                <!VAL_REASSIGNMENT!>a<!> = "12"
```

Comment:
This should be captured val initialization, which means the current captured writes detector misses something... Anyway, reporting a different diagnostic would be okay-ish. :\